holidays/countries/saint_vincent_and_the_grenadines.py

```
@@ -0,0 +1,99 @@
+
+        # Independence Day.
+        self._add_holiday_oct_27(tr("Independence Day"))
+
+        # Christmas Day.
+        self._add_christmas_day(tr("Christmas Day"))
+
+        # Boxing Day.
+        self._add_christmas_day_two(tr("Boxing Day"))
+
+
+class VC(SaintVincentAndTheGrenadines):
```

Pylint (3.3.7)

[convention] Missing class docstring

(C0115)